Project-I-Social-Card/social-card/src/components/CardComponents/CardBanner.js

```diff
@@ -1,2 +1,11 @@
 import React from 'react';
 import './Card.css';
+
+const CardBanner = ({url, alt})=>{
```

Comment:
Good job on deconstructing the props object here, there is a convention though to leave an empty space before and after your properties.